### PR TITLE
app-text/dvisvgm: Bump to version 2.9.1

### PR DESCRIPTION
**Pull request by mechpen**
Signed-off-by: Wenhua Zhao <whzhao@gmail.com>

### CODE REVIEW

File: app-text/dvisvgm/dvisvgm-2.9.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic
```
Comment (Zlogene) [inline app-text/dvisvgm/dvisvgm-2.9.1.ebuild:6]: none of these eclasses are used.

File: app-text/dvisvgm/dvisvgm-2.9.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	>=app-arch/brotli-1.0.5
+	app-text/ghostscript-gpl
+	dev-libs/kpathsea:=
+	>=dev-libs/xxhash-0.6.5
+	>=media-gfx/potrace-1.10-r1
+	media-libs/freetype:2
+	>=media-libs/woff2-1.0.2
+	dev-libs/kpathsea
+	sys-libs/zlib"
+DEPEND="${RDEPEND}
```
Comment (Zlogene) [inline app-text/dvisvgm/dvisvgm-2.9.1.ebuild:33]: these all are `BDEPEND`

File: app-text/dvisvgm/dvisvgm-2.9.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# TODO unbundle app-arch/brotli
+RDEPEND="virtual/tex-base
+	>=app-arch/brotli-1.0.5
+	app-text/ghostscript-gpl
+	dev-libs/kpathsea:=
+	>=dev-libs/xxhash-0.6.5
+	>=media-gfx/potrace-1.10-r1
+	media-libs/freetype:2
+	>=media-libs/woff2-1.0.2
+	dev-libs/kpathsea
```
Comment (Zlogene) [inline app-text/dvisvgm/dvisvgm-2.9.1.ebuild:31]: double kpathsea dep.

File: app-text/dvisvgm/dvisvgm-2.9.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE="test"
+# Tests don't work from $WORKDIR: kpathsea tries to search in relative
+# directories from where the binary is executed.
+# We cannot really use absolute paths in the kpathsea configuration since that
+# would make it harder for prefix installs.
+RESTRICT="test"
+
+# TODO unbundle app-arch/brotli
+RDEPEND="virtual/tex-base
```
Comment (Zlogene) [inline app-text/dvisvgm/dvisvgm-2.9.1.ebuild:23]: switch to the texlive-core directly.

**Comment by mechpen** [comment]
Thank you for reviewing! I fixed these accordingly.

**Comment by ajakk** [comment]
Looks like this has been bumped in the meantime. Sorry no one got to this sooner!

**Pull request closed without merging**